pkg/test/context.go

```diff
@@ -52,38 +56,37 @@ func NewTestCtx(t *testing.T) *TestCtx {
 			"test",
 		)
 	} else {
-		prefix = "main"
+		prefix = "operator-sdk"
 	}
 
-	id := prefix + "-" + strconv.FormatInt(time.Now().Unix(), 10)
+	// add a creation timestamp to the ID
+	id := prefix + "-" + strconv.FormatInt(time.Now().UnixNano(), 10)
```

Comment:
What is the reason for switching to `UnixNano`?

Reply:
To prevent collisions. Collisions _shouldn't_ have happened before due to the namespace being based on the name of the test, but now it could be more likely to happen if users run create multiple contexts in parallel since those would use the same name and potentially be created in the same second. Using nanoseconds should prevent collisions.